src/linker/wasm.rs

```diff
@@ -29,14 +29,11 @@ pub fn link(input: &[u8], name: &str) -> Vec<u8> {
         CString::new("--global-base=0").unwrap(),
     ];
 
-    command_line.push(CString::new("--export").unwrap());
-    command_line.push(CString::new("deploy").unwrap());
-    command_line.push(CString::new("--export").unwrap());
-    command_line.push(CString::new("call").unwrap());
+    command_line.push(CString::new("--export-dynamic").unwrap());
```

Comment:
Since the exported functions are dynamic, I resorted to using `--export-dynamic`, although in a proper implementation maybe it would be better to retain more control over this and iterate over the list of functions with `--export` flags.